entrypoint.sh

```diff
@@ -2,6 +2,32 @@
 
 set -eu
 
+function get_diff() {
+  base="$1"
+  sha="$2"
+  filter="$3"
+  mapfile -t submodules < <(git submodule | awk '{print $2}')
```

Comment:
There also seems to be an issue with using mapfile on Mac which fails silently

See: https://github.com/tj-actions/changed-files/runs/5290697419?check_suite_focus=true#step:3:128

```
All modified files: 
[121](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:121)
  awk: i/o error occurred while closing /dev/stdout
[122](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:122)
   input record number 2, file 
[123](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:123)
   source line number 1
[124](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:124)
  awk: i/o error occurred while closing /dev/stdout
[125](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:125)
   input record number 2, file 
[126](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:126)
   source line number 1
[127](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:127)
  awk: i/o error occurred while closing /dev/stdout
[128](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:128)
   input record number 2, file 
[129](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:129)
   source line number 1
[130](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:130)
  awk: i/o error occurred while closing /dev/stdout
[131](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:131)
   input record number 2, file 
[132](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:132)
   source line number 1
[133](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:133)
awk: i/o error occurred while closing /dev/stdout
[134](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:134)
 input record number 2, file 
[135](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:135)
 source line number 1
[136](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:136)
awk: i/o error occurred while closing /dev/stdout
[137](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:137)
 input record number 2, file 
[138](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:138)
 source line number 1
[139](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:139)
awk: i/o error occurred while closing /dev/stdout
[140](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:140)
 input record number 2, file 
[141](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:141)
 source line number 1
[142](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:142)
awk: i/o error occurred while closing /dev/stdout
[143](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:143)
 input record number 2, file 
[144](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:144)
 source line number 1
[145](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:145)
awk: i/o error occurred while closing /dev/stdout
[146](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:146)
 input record number 2, file 
[147](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:147)
 source line number 1
[148](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:148)
awk: i/o error occurred while closing /dev/stdout
[149](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:149)
 input record number 2, file 
[150](https://github.com/tj-actions/changed-files/runs/5290952730?check_suite_focus=true#step:3:150)
 source line number 1
```

Reply:
Looks like I need to target the brain-dead OSX bash. Okay.